widgetssdk/src/main/java/com/glia/widgets/view/head/ActivityWatcherForChatHead.kt

```diff
@@ -67,4 +141,45 @@ internal class ActivityWatcherForChatHead(
         val configuration = Dependencies.getSdkConfigurationManager()?.createWidgetsConfiguration()
         return Configuration.Builder().setWidgetsConfiguration(configuration)
     }
+
+    private fun navigateToChat(activity: Activity?) {
+        activity?.let {
+            val contextWithStyle = MaterialThemeOverlay.wrap(
```

Comment:
no need to wrap context with `MaterialThemeOverlay` because we're using it only to start the `Activity`